File: Python/A_NivelIniciante/Exercicio_3037.py
n = int(input())
lancamento = []
for x in range(n):
	total1 = 0
	total2 = 0
	for z in range(3):
		lancamento = input().split(" ")
		total1+= int(lancamento[0]) * int(lancamento[1])
	for z in range(3):
		lancamento = input().split(" ")
		total2+= int(lancamento[0]) * int(lancamento[1])
	print("JOAO" if total1>total2 else "MARIA")

File: Javascript/A_NivelIniciante/Exercicio_3037.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var n = lines.shift()*1;
for(let i = 0;i<n;i++){
    var total1 = 0;
    var total2 = 0;
    for (let z = 0;z<3;z++){
    var lancamento = lines.shift().split(" ");
    total1+= lancamento[0]*1 * lancamento[1]*1;
    }
    for (let z = 0;z<3;z++){
    var lancamento = lines.shift().split(" ");
    total2+= lancamento[0]*1 * lancamento[1]*1 ;
    }
    console.log(total1>total2?"JOAO":"MARIA");
}